rt_gene/src/rt_gene/__init__.py

```diff
@@ -1,8 +1,2 @@
-from . import estimate_gaze_base
```

Comment:
Not sure why they were removed - it's nice to have them in my opinion.

Reply:
sorry, that was a rebasing issue, not intentional. Thank you for catching.